tools/RAiDER/aria/calcARIA.py

```diff
@@ -0,0 +1,29 @@
+"""
+Calculate the interferometric phase from the 4 delays files of a GUNW
```

Comment:
Find the 4 delays confusing to a reader.

Calculate the interfer.... phase delay at acquisition time reported in the GUNW product.

Reply:
Will make sure its clear in #441